docker/junit-rule/src/main/java/org/arquillian/cube/docker/junit/Reflections.java

```diff
@@ -0,0 +1,56 @@
+package org.arquillian.cube.docker.junit;
+
+
+import java.lang.reflect.Field;
+import java.lang.reflect.ParameterizedType;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class Reflections {
+
+    public static final void injectObject(Object instance, Field field, Object injection) throws IllegalAccessException {
```

Comment:
static method can't be overridden, as they are associated with class. Do we have any reason to make it final

Reply:
https://stackoverflow.com/questions/1743715/behaviour-of-final-static-method

Reply:
@lordofthejars What's the purpose here to make static method final?

Reply:
The keyword final will disable the method from being hidden. So they cannot be hidden and an attempt to do so will result in a compiler error. `final` modifier is just a compilation time modifier it is dropped in code generation phase, so it is just a helper operator for developers. In this case if anybody tries to extend reflections class and `create` a method with same name, compiler will warn (error) saying that you are doing something that might confuse other people when read the code.

Reply:
thx for explanation. Now this is clear to me